### ISSUE TITLE
Rexray 0.3.3 config file not compatible with Rexray 0.4 

### ISSUE THREAD

**Issue by VibhorGupta** [emccode/rexray#471]
Rexray 0.3.3 config file is not compatible with Rexray 0.4. 

I have 2 instances of rexray 0.3.3 running fine. I pulled rexray 0.4 on a new server and used the same config file as on the 0.3.3.

rexray 0.3.3 config file example:

```
rexray:
storageDrivers:
- scaleio
volume:
 mount:
  preempt: true
scaleio:
endpoint: xxxxxxxxx
insecure: true
userName: admin
password: xxxxxxxxx
systemID: 0
protectionDomainName: pd1
storagePoolName: hdd
```

Rexray 0.4 was dying with the following error:

```

rexray service start -l debug
DEBU[0000] updated log level                             logLevel=debug
DEBU[0000] os.args                                       time=1466613572 val=[rexray service start -l debug]
DEBU[0000] invoking service start                        os.Args=[rexray service start -l debug] time=1466613572
DEBU[0000] starting in background                        time=1466613572
Starting REX-Ray...DEBU[0000] trying to start service                       client=/var/run/rexray/client.sock time=1466613572
DEBU[0000] accepted connection                           time=1466613573
DEBU[0000] received data                                 time=1466613573
FAILED!
DEBU[0000] exiting with error code 1
```

temporary solution is to use the following config file:

```
libstorage:
  service: scaleio
  integration:
    volume:
      operations:
        mount:
          preempt: true
scaleio:
  endpoint: xxxxxxxxxx
  insecure: true
  userName: admin
  password: xxxxxxxxxx
  systemID: 0
  protectionDomainName: pd1
  storagePoolName: hdd
```

it would be good to use the old rexray.storagedrivers as default if libstorage.service is not defined.

thanks

**Comment by akutz** [comment]
Thank you for taking the time to file this @vibhorgupta! I'm glad we got you online again so quickly. I'll definitely mark this patch a priority and ensure it's included in 0.4.1.

**Comment by akutz** [comment]
Hi @VibhorGupta, I tested #481 and showed it solves this, so I'm going to close this issue. Look for this fix in the next release.
